gui/gui.md

```diff
@@ -69,10 +63,20 @@ As with `HostnameWindow`, the hostname is validated against RFC 1123.
 
 If any field is invalid, `Apply` will be disabled.
 
+## Log Window
```

Comment:
Describe what sort of messages to expect in the logs.

Generally, this file needs a re-write